pkg/crc/machine/cert-renewal.go

```diff
@@ -0,0 +1,229 @@
+package machine
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"time"
+
+	"github.com/code-ready/crc/pkg/crc/errors"
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/code-ready/crc/pkg/crc/oc"
+	"github.com/code-ready/crc/pkg/crc/ssh"
+	"github.com/code-ready/crc/pkg/crc/systemd"
+	"github.com/code-ready/crc/pkg/crc/version"
+)
+
+var (
+	releaseImage = fmt.Sprintf("quay.io/openshift-release-dev/ocp-release:%s", version.GetBundleVersion())
```

Comment:
@cfergeau Are will still using this, since with new bundle we already have `openshift/cert-recovery` tag?

Reply:
Yes, I haven't tested using `openshift/cert-recovery` yet, so I'd favour sticking with this for now, and switching to `openshift/cert-recovery` in a follow-up PR after the release.